### PR TITLE
Improved Haskell syntax highlighting

### PR DESCRIPTION
**Pull request by Brettm12345**
I also added a `yarn.lock` file and a prettier configuration.

# Screenshots

<details>
<summary>Type Signatures</summary>

haskell

</details>

<details>
<summary>Imports</summary>

imports

</details>

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "printWidth": 100
```
Comment (atomiks) [inline .prettierrc]: Is there a reason for using 100 over the default 80?

Comment (Brettm12345) [reply .prettierrc]: Yeah I set it up to closer match the initial formatting

Comment (atomiks) [reply .prettierrc]: I think I already had it prettier'd on save with my global settings which has it at 80. I see some arrays have changed formatting — can we just leave it at 80 / default settings?

**Comment by Brettm12345** [comment]
> I think I already had it prettier'd on save with my global settings which has it at 80. I see some arrays have changed formatting — can we just leave it at 80 / default settings? 

Done

**Comment by atomiks** [comment]
Thanks

**Pull request merged**